webapp/components/Editor/nodes/Hashtag.js

```diff
@@ -21,11 +21,15 @@ export default class Hashtag extends TipTapMention {
     return {
       ...super.schema,
       toDOM: node => {
+        // use a dummy domain because URL cannot handle relative urls
+        const url = new URL('/', 'http://example.org')
```

Comment:
shouldn't we just use something else that can handle it then?

Reply:
actually, looking at the docs, and what you are actually doing here, if the only thing you are using is `url.search` then you could just use our domain instead of a dummy one

Reply:
Yes @mattwr18 , I would like to see our domain here as well! 😘

Reply:
What will be the network domain at the end?
`https://human-connection.org` is used for the donation and information page at the moment.
But I decided to change it to this. Hope it is okay for you @roschaefer !?